internal/custom_executable/head/head.go

```
@@ -0,0 +1,358 @@
+  -q, --quiet          never print headers giving file names
+  -v, --verbose        always print headers giving file names
+  --help               display this help and exit
+  --version            output version information and exit`
+)
+
+type options struct {
+	lineCount   int
+	byteCount   int
+	quiet       bool
+	verbose     bool
+	invertCount bool
```

golangci-lint (1.64.8)

field `invertCount` is unused

(unused)